src/screen/Council/CouncilScreen.tsx

```diff
@@ -132,17 +175,200 @@ function Item({
   );
 }
 
+type CouncilVoteProps = {
+  visible: boolean;
+  setVisible: (visible: boolean) => void;
+  candidates: string[];
+  module: string | null;
+};
+
+function CouncilVote({visible, setVisible, candidates, module}: CouncilVoteProps) {
+  const [account, setAccount] = React.useState<string>();
+  const [amount, setAmount] = React.useState<string>('');
+  const [selectedCandidates, setSelectedCandidates] = React.useState<Array<string>>([]);
+
+  const {api} = useApi();
+  const startTx = useApiTx();
+  const {networkAccounts} = useAccounts();
+  const formatBalance = useFormatBalance();
+
+  const onCandidateSelect = (accountId: string, isSelected: boolean) => {
```

Comment:
probably something like this would work also, https://lodash.com/docs/4.17.15#xor
```suggestion
  const onCandidateSelect = (accountId: string, isSelected: boolean) => xor(selectedCandidates, [accountId])
```

Reply:
This could work but here we have an extra logic. The selection should stop once `MAX_VOTES` is reached.